examples/webgl_lights_spotlight.html

```diff
@@ -161,7 +162,8 @@
 					distance: spotLight.distance,
 					angle: spotLight.angle,
 					penumbra: spotLight.penumbra,
-					decay: spotLight.decay
+					decay: spotLight.decay,
+					'shadow intensity': spotLight.shadow.intensity,
```

Comment:
Seems like if you're adding shadow intensity to the controls, then the other shadow params should be there too, at least `spotLight.shadow.camera.near` and `spotLight.shadow.camera.far`

Reply:
I tried it, but I don't see any benefits from changing near plane for example. What do you think?
1. It clutters control interface
2. Far plane is somehow overridden by light's `distance` parameter
3. Don't see any case when it useful in this particular example

Reply:
> Far plane is somehow overridden by light's distance parameter

Yes, the `spotLight.shadow.camera.far` has no effect unless  `spotLight.distance = 0`. Demonstrating that is actually the main reason why I want to add `near` and `far` to this example, since it's something that we should make clearer.

Reply:
I've added two parameters. Please, check 🤖

Reply:
Looks good, although could you also change the minimum value for `.distance` to 0?